### ISSUE TITLE
Clipboard indicator for Ubuntu Touch

### ISSUE THREAD

**Issue by thilovv** [ubports/ubuntu-touch#1148]
Feature request:
I use the 'clipboard indicator' a lot on my PC (xfce) and it would be very helpful for me having it on Ubuntu Touch, too.

**Comment by Fuseteam** [comment]
can you elaborate how the "clipboard indicator" works?

**Comment by thilovv** [comment]
I mean a clipboard manager like those:
https://www.tecmint.com/best-clipboard-managers-for-linux/

**Comment by Fuseteam** [comment]
if you want you can rename the issue to "clipboard manager" instead of indicator as some devs are not keen on adding more indicators but may be inclined to look at ways to get a clipboard manager working in UT

**Comment by PhoenixLandPirate** [comment]
> not keen on adding more indicators

I agree with this, HOWEVER, we do have an "Text Entry Setting's", perhaps we could add it to that indicator?

**Comment by Fuseteam** [comment]
That's related to the physical keyboard currently
i wonder if we can add it to the osk

**Comment by PhoenixLandPirate** [comment]
Could maybe fit it somewhere on the keyboard, could either be something when you hold on the space bar, or the "Switch input" button thats meant for languages, but I dont think emoji's are a language, so you could maybe put a clipboard manager in there too.

**Comment by Fuseteam** [comment]
I think it could fit in the cursor mode
